phpcs.xml.dist

```diff
@@ -35,18 +39,36 @@
     <rule ref="Generic.Strings.UnnecessaryStringConcat.Found">
         <exclude-pattern>lib/Doctrine/Migrations/MigrationsVersion.php</exclude-pattern>
     </rule>
+    <rule ref="Squiz.Strings.DoubleQuoteUsage.ContainsVar">
+        <exclude-pattern>tests/Doctrine/Migrations/Tests/TestLogger.php</exclude-pattern>
+    </rule>
 
     <rule ref="SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod">
         <exclude-pattern>lib/Doctrine/Migrations/Configuration/AbstractFileConfiguration.php</exclude-pattern>
     </rule>
 
+    <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint">
+        <exclude-pattern>tests/Doctrine/Migrations/Tests/TestLogger.php</exclude-pattern>
+    </rule>
+
+    <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint">
```

Comment:
What's the reason to exclude all these files?

Reply:
Fixed. Only `lib/Doctrine/Migrations/Version/Version.php` is needed because of __toStirng method